os/os-mac.h

```diff
@@ -117,3 +122,60 @@ static inline bool os_cpu_has(cpu_features feature)
 }
 
 #endif
+
+#if defined (__APPLE__)
```

Comment:
You won't need this guard here as `os-mac.h` will only be included on macOS (see https://github.com/axboe/fio/blob/58818df7b0948d36ba9743160a1f570700d2347d/os/os.h#L49 ).